connectors/sources/onedrive.py

```diff
@@ -211,13 +340,126 @@ async def close(self):
     async def ping(self):
         """Verify the connection with OneDrive"""
         try:
-            await anext(self.get_client.get(url_name=ENDPOINTS["PING"]))
+            url = parse.urljoin(BASE_URL, ENDPOINTS[PING])
+            await anext(self.get_client.get(url=url))
             self._logger.info("Successfully connected to OneDrive")
         except Exception:
             self._logger.exception("Error while connecting to OneDrive")
             raise
 
+    def _pre_checks_for_get_content(
+        self, attachment_extension, attachment_name, attachment_size
+    ):
+        if attachment_extension == "":
+            self._logger.debug(
+                f"Files without extension are not supported, skipping {attachment_name}."
+            )
+            return
```

Comment:
Can we return `False` explicitly?

Reply:
Good call, Done here and all other relevant places